app-arch/unzip/unzip-6.0_p27-r1.ebuild

```diff
@@ -64,13 +64,11 @@ src_configure() {
 	case ${CHOST} in
 		i?86*-*linux*)       TARGET="linux_asm" ;;
 		*linux*)             TARGET="linux_noasm" ;;
-		i?86*-*bsd* | \
-		i?86*-dragonfly*)    TARGET="freebsd" ;; # mislabelled bsd with x86 asm
+		i?86*-*bsd* | i?86*-dragonfly*) TARGET="freebsd" ;; # mislabelled bsd with x86 asm
```

Comment:
trash it? like we care about prefix on dragonflybsd

Reply:
ok